charts/bitbucket-runner-autoscaler/templates/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
I think we can leave the creation of the namespace to Helm engine, or at least make it optional.